src/EditorFeatures/Test/CommentSelection/CommentUncommentSelectionCommandHandlerTests.cs

```diff
@@ -497,6 +499,160 @@ public void Uncomment_BlockCommentWithNoEnd()
             UncommentSelection(code, Enumerable.Empty<TextChange>(), new Span(8, 0), supportBlockComments: true);
         }
 
+        [WpfFact, Trait(Traits.Feature, Traits.Features.CommentSelection)]
+        [WorkItem(31669, "https://github.com/dotnet/roslyn/issues/31669")]
+        public void Uncomment_BlockWithSingleInside()
+        {
+            var code = @"
+class A
+{
+    |start|/*
+    void M()
+    {
+            // A comment
+            // Another comment
+    }
+    */|end|
```

Comment:
❗️  Need to include a test which matches the behavior of line selection in the column. In such a case, `|end|` will be the start of the line after `*/`.

Reply:
Something like this?

```csharp
class A
{
|start|
    /*
    void M()
    {
            // A comment
            // Another comment
    }
    */
|end|
}
```

This does work since the span is trimmed, but will add a test for it as well.

Reply:
I was thinking this:

```csharp
{
|start|    /*
  …
    */
|end|}
```

But both would be good.